sound/soc/sof/intel/hda.h

```diff
@@ -414,7 +414,7 @@
 	(HDA_DSP_BDL_SIZE / sizeof(struct sof_intel_dsp_bdl))
 
 /* Number of DAIs */
-#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
```

Comment:
commit message mismatch repeated

Reply:
ah yes, fixed now

Reply:
I would have spelled out that the number of DAIs were based on CONFIG_SND_SOC_SOF_HDA while the declaration is using CONFIG_SND_SOC_SOF_HDA_AUDIO_CODEC, but that's just a nitpick.

Reply:
I left the patch as is, it's just confusing to explain this Kconfig discrepancy.